docs/design/coordinator.md

```diff
@@ -113,7 +113,7 @@ need compaction.
 A set of segments needs compaction if all conditions below are satisfied:
 
 1) Total size of segments in the time chunk is smaller than or equal to the configured `inputSegmentSizeBytes`.
-2) Segments have never been compacted yet or compaction spec has been updated since the last compaction, especially `maxRowsPerSegment`, `maxTotalRows`, and `indexSpec`.
+2) Segments have never been compacted yet or compaction spec has been updated since the last compaction: `maxTotalRows` or `indexSpec`.
```

Comment:
I know this isn't part of the change but we have added a bunch of other specs to auto compaction that if changed, will trigger compaction (i.e. dimensionSpec, metricsSpec, etc.)